code/registration.py

```diff
@@ -8,48 +8,39 @@
 import tempfile
 import warnings
 from datetime import datetime as dt
-from functools import partial
+from functools import lru_cache, partial
 from glob import glob
 from itertools import product
 from multiprocessing.pool import ThreadPool
 from pathlib import Path
 from time import time
-from typing import Callable, Optional, Tuple, Union, List
-from functools import lru_cache
+from typing import Callable, List, Optional, Tuple, Union
 
 import cv2
 import h5py
 import matplotlib as mpl
 import numpy as np
 import pandas as pd
 import suite2p
-from aind_data_schema.core.processing import (
-    DataProcess,
-    PipelineProcess,
-    Processing,
-    ProcessName,
-)
+from aind_data_schema.core.processing import DataProcess
+from aind_data_schema.core.quality_control import QCMetric, QCStatus, Status
+from aind_data_schema_models.process_names import ProcessName
```

Comment:
is that the same as importing both from aind_data_schema, as you do elsewhere?
`from aind_data_schema.core.processing import DataProcess, ProcessName`

Reply:
It should result in the same, since aind_data_schema.core.processing imports ProcessName from aind_data_schema_models itself.

I'd prefer this importing from aind_data_schema_models explicitly here, since it didn't exist in the aind_data_schema version this is upgrading from (0.22.0)